docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -112,35 +127,6 @@ Use the following procedure to enable the feature to use an OIDC Access Token as
 
 ## Troubleshooting
```

Comment:
I think we can refactor the current ones and mention that failure can occur if:
- JWK is not available or incorrectly configured.
- JWK was revoked by the authorization server and the update has not happened yet in the api ml (in this case configure better value for the refresh interval or restart api ml, or just wait)